cms/envs/production.py

```diff
@@ -697,3 +697,14 @@ def get_env_setting(setting):
 # .. for now it wil impact country listing in auth flow and user profile.
 # .. eg ['US', 'CA']
 DISABLED_COUNTRIES = ENV_TOKENS.get('DISABLED_COUNTRIES', [])
+
+# .. setting_name: LIBRARY_ENABLED_BLOCKS
+# .. setting_default: ['problem', 'video', 'html', 'drag-and-drop-v2']
+# .. setting_description: List of block types that are enabled to be created/used
+# .. in libraries. Both basic blocks and advanced blocks can be included.
+# .. In the future, advanced block configuration will be handled individually per library.
+# .. eg ['problem', 'video', 'drag-and-drop-v2]
+LIBRARY_ENABLED_BLOCKS = ENV_TOKENS.get(
+    'LIBRARY_ENABLED_BLOCKS',
+    ['problem', 'video', 'html', 'drag-and-drop-v2', 'google-calendar', 'google-document'],
+)
```

Comment:
```suggestion
```
Why does production.py need a default beyond the default that common.py already declares? If google-calendar and google-document are reasonable items to have in the defaults list, then pls include them in the common.py list as well.

Reply:
Sorry, that was a mistake, we should not add `google-document` or `google-calendar` yet, it was deleted here: https://github.com/openedx/edx-platform/commit/057492327404071cecc1d4d9178efacb7cc79636